content/docs/command-reference/remote/modify.md

```diff
@@ -586,16 +586,31 @@ for a full guide on using Google Drive as DVC remote storage.
   ```
 
 - `gdrive_user_credentials_file` - path where DVC stores OAuth credentials to
-  access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
+  access Google Drive data. `$CACHE_HOME/pydrive2fs/$client_id/default.json` by
+  default, where `CACHE_HOME` is:
+
+  | macOS location     | Linux location (typical\*) | Windows location        |
+  | ------------------ | -------------------------- | ----------------------- |
+  | `~/Library/Caches` | `~/.cache/`                | `%CSIDL_LOCAL_APPDATA%` |
+
+  Also, see `profile` config option below.
 
   ```dvc
   $ dvc remote modify myremote \
-        gdrive_user_credentials_file .dvc/tmp/mycredentials.json
+        gdrive_user_credentials_file /path/to/mycredentials.json
   ```
 
 See [Authorization](/doc/user-guide/setup-google-drive-remote#authorization) for
 more details.
 
+- `profile` - credentials cache profile name. By default, credentials will be
+  cached per `client_id`, but if you are using multiple users you might want to
+  use ‘profile’ to avoid accidentally using someone else’s cached credentials.
```

Comment:
```suggestion
- `profile` - name for the credentials cache (part of the default
  `gdrive_user_credentials_file` path). By default, credentials will be cached
  per `gdrive_client_id `, but if you have multiple system users, you can set a
  custom ‘profile’ name to avoid using someone else’s credentials accidentally.
```